_src/edx-pl/sass/components/_colors.scss

```diff
@@ -61,13 +61,19 @@
 .swatch-meta {
     padding: spacing-vertical(xx-small) spacing-horizontal(small);
     white-space: nowrap;
-    font-size: font-size(small);
+    font-size: font-size(x-small);
     @include text-align(center);
     color: palette(grayscale, dark);
 
     .color-rgb {
```

Comment:
While we're here, let's make it easier for folks to copy these (a la the git branches in the github UI - screengrab below), I'd love to see us be able to double/triple click to select each RGB or hex value for easy copying.

![easy-copying-github](https://cloud.githubusercontent.com/assets/163763/7985352/20e82790-0a9e-11e5-820a-5df52b0ba173.png)

Mind trying to add this as well?